source/en/users_user_types.md

```diff
@@ -38,8 +36,12 @@ Group Admin Users only have access to the parts of the CMS as assigned to the Us
 
 A User will only have access to the parts of the CMS as assigned to the User Group they belong to as well as access to their own Media, Layouts and Schedules.
 
-{tip}
+### DOOH
 
-User Type is selected when first adding a User and can be edited from the individual User record.
+A DOOH user will have access to the CMS, via the APi, to allow for content creation and Scheduling and will only see Displays / Media / Playlists / Layouts that belong to them.
```

Comment:
Should we also say that content (displays/media/layouts/schedules/etc) which are created by the DOOH user are not visible by other non-DOOH users. And that a Super User can go into their profile and set that they also want to see the DOOH users content?